src/main/java/juja/sqlcmd/DataSet.java

```diff
@@ -0,0 +1,40 @@
+package juja.sqlcmd;
+
+import java.util.Arrays;
+
+public class DataSet {
+
+    private String[] oneRow;
```

Comment:
Is it essential to specify number in varibale name?

Reply:
i will renaming this varibale.